### PR TITLE
feat: add option to disable insurance for shipments to Belgium

### PR DESCRIPTION
**Pull request by Mark-Ernst**
It is not possible to open a pull request. Make a fork to keep your own version.

### CODE REVIEW

File: includes/admin/OrderSettings.php [hunk 1]
```diff
@@ -12,6 +13,7 @@ class OrderSettings
 {
     private const DEFAULT_COLLO_AMOUNT = 1;
     private const FIRST_INSURANCE      = 1;
+    public const DEFAULT_BELGIAN_INSURANCE = 500;
```
Comment (EdieLemoine) [inline includes/admin/OrderSettings.php]: uitlijnen

File: includes/admin/OrderSettings.php [hunk 2]
```diff
@@ -361,6 +365,11 @@ private function setInsuranceData(): void
             $insuranceAmount = $this->getCarrierSetting(WCPOST_Settings::SETTING_CARRIER_DEFAULT_EXPORT_INSURED_AMOUNT);
         }
 
+        if ($isDefaultInsured && $isBelgium) {
+            $isInsured = $isDefaultInsuredForBE;
```
Comment (EdieLemoine) [inline includes/admin/OrderSettings.php]: uitlijnen

File: includes/admin/OrderSettingsRows.php [hunk 3]
```diff
@@ -130,6 +131,25 @@ public static function getOptionsRows(
             $rows = array_merge($rows, self::getAdditionalOptionsRows($orderSettings));
         }
 
+        if ($isBelgium) {
+            $rows[] = [
```
Comment (EdieLemoine) [inline includes/admin/OrderSettingsRows.php:135]: single quotes

File: includes/admin/settings/class-wcpost-settings.php [hunk 4]
```diff
@@ -88,7 +88,7 @@ class WCPOST_Settings
     public const SETTING_CARRIER_DEFAULT_EXPORT_INSURED            = "export_insured";
     public const SETTING_CARRIER_DEFAULT_EXPORT_INSURED_AMOUNT     = "export_insured_amount";
     public const SETTING_CARRIER_DEFAULT_EXPORT_INSURED_FROM_PRICE = "export_insured_from_price";
-
+    public const SETTING_CARRIER_DEFAULT_EXPORT_INSURED_FOR_BE           = 'export_insured_for_be';
```
Comment (EdieLemoine) [inline includes/admin/settings/class-wcpost-settings.php]: uitlijnen

**Review by EdieLemoine — CHANGES_REQUESTED** [review]

**Review by joerivanveen — APPROVED** [review]

**Review by RichardPerdaan — APPROVED** [review]

**Pull request merged**